packages/common/src/hooks/useFormContext/useFormContext.ts

```diff
@@ -0,0 +1,18 @@
+import create from 'zustand';
+import { Item } from '../..';
+
+type FormContext = {
+  item: Item;
+  setItem: (item: Item) => void;
+};
+
+export const useFormContext = create<FormContext>(set => ({
+  item: {
```

Comment:
you had used a format for defining defaults for objects that I liked, and realise now that I forgot to do for this one!

Reply:
ah. part of the version prior to `react-hook-form` removed in 98bd632